File: Assignment1/factorial.java
import java.util.*;

public class factorial
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter number: ");
		int n = sc.nextInt();
		int fact=1;
		while (n>0)
		{
			fact *= n;
			n--;
		}
		while (fact>9)
		{
			fact /= 10;
		}
		System.out.println(fact);

		sc.close();
	}
}

File: Assignment1/factorial.cpp
#include <iostream>
using namespace std;

int main() {
    int n, fact = 1;
    cout << "Enter number: ";
    cin >> n;
    while (n > 0) {
        fact *= n;
        n--;
    }
    while (fact > 9) {
        fact /= 10;
    }
    cout << fact << endl;
    return 0;
}